bedrock/firefox/forms.py

```diff
@@ -10,8 +10,6 @@
 
 from lib.l10n_utils.dotlang import _lazy as _
 
-LANG_FILES = ['firefox/whatsnew-fx37']
-
```

Comment:
Looks like it was [translated for at least some locales](https://github.com/mozilla-l10n/www.mozilla.org/commit/06c998d2d46dcaf42afcae61502dd9593cc41b68#diff-399337a0fd4833ee8c0b111db134fe73L73).